inginious/frontend/pages/course_admin/statistics.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+
+""" Utilities for computation of statistics  """
+
+def compute_statistics(task, data):
+    """ 
+    Compute statistics about submissions and tags.
+    This function returns a tuple of lists following the format describe below:
+    (   
+        [('Number of submissions', 13), ('Evaluation submissions', 2), …], 
+        [(<tag>, '61%', '50%'), (<tag>, '76%', '100%'), …]
+    )
+     """
+    total_submission = len(data)
+    total_submission_best = 0
+    total_submission_best_succeeded = 0
+        
+    for submission in data:
+        if "best" in submission and submission["best"]:
+            total_submission_best = total_submission_best + 1
+            if "result" in submission and submission["result"] == "success":
+                total_submission_best_succeeded += 1
+        
+    statistics = []
```

Comment:
Don't be afraid to put the code on several lines rather than calling ``append`` several times